From 5830cc5b0cd3fd7f3ad86824dc718b4cfc1d4117 Mon Sep 17 00:00:00 2001
Author: Simon Ruderich <simon@ruderich.org>
Bug-Debian: http://bugs.debian.org/664086
Subject: pass BUILD_CFLGAS and BUILD_LDFLAGS to menuselect

Allow menuselect to get its set of CFLAGS and LDFLAGS through the
environment of Make:

  make BUILD_CFLAGS="whatever" BUILD_LDFLAGS="whatever"

Also keep menuselect from steping over them on its own.

Commited to upstream trunk (asterisk r366002, menuselect r1033).
Will be included in asterisk 11.

---
 Makefile |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -845,7 +845,9 @@ nmenuselect: menuselect/nmenuselect menu
 	-@menuselect/nmenuselect menuselect.makeopts && (echo "menuselect changes saved!"; rm -f channels/h323/Makefile.ast main/asterisk) || echo "menuselect changes NOT saved!"
 
 # options for make in menuselect/
-MAKE_MENUSELECT=CC="$(BUILD_CC)" CXX="" LD="" AR="" RANLIB="" CFLAGS="" $(MAKE) -C menuselect CONFIGURE_SILENT="--silent"
+MAKE_MENUSELECT=CC="$(BUILD_CC)" CXX="" LD="" AR="" RANLIB="" \
+		CFLAGS="$(BUILD_CFLAGS)" LDFLAGS="$(BUILD_LDFLAGS)" \
+		$(MAKE) -C menuselect CONFIGURE_SILENT="--silent"
 
 menuselect/menuselect: menuselect/makeopts
 	+$(MAKE_MENUSELECT) menuselect
